README.md

```diff
@@ -74,6 +74,7 @@ MediumEditor also supports textarea. If you provide a textarea element, the scri
 * __allowMultiParagraphSelection__: enables the toolbar when selecting multiple paragraphs/block elements. Default: true
 * __delay__: time in milliseconds to show the toolbar or anchor tag preview. Default: 0
 * __disableAnchorPreview__: enables/disables the anchor preview element, which appears when hovering links and allows the user to edit the link when clicking. If toolbar is diabled (via __disableToolbar__ or `data-disable-toolbar attribute`) the anchor preview is always disabled so this option will be ignored. Default: false
+* __enableAutoLinker__: enables/disables the auto-linker feature, which automatically turns URLs entered into the text field into HTML anchor tags (similar to the functionality of Markdown). Default: false
```

Comment:
I would call this `autoLink` so that in the future, if we add options for the auto link extension, we can just nest them inside the options:

ie:
```js
{
  autoLink: {
    option1: true,
    option2: 'something'
  }
}
```

or you can enable it like this:
```js
{
  autoLink: true
}
```

and the default options would still be valid:
```js
{
  autoLink: false
}
```